integration_tests/backend/conftest.py

```diff
@@ -0,0 +1,20 @@
+import os
+
+import pytest
+
+import aqueduct
+
+API_KEY_ENV_NAME = "API_KEY"
+SERVER_ADDR_ENV_NAME = "SERVER_ADDRESS"
+ADAPTER_ENV_NAME = "ADAPTER"
+
+
+def pytest_configure(config):
+    pytest.api_key = os.getenv(API_KEY_ENV_NAME)
+    pytest.server_address = os.getenv(SERVER_ADDR_ENV_NAME) or "localhost:8080"
+    pytest.adapter = os.getenv(ADAPTER_ENV_NAME) or "http://"
```

Comment:
I think we should just error if the env variables aren't supplied. Let's not hide the test configuration if we can.

Reply:
Why not force the user to put the full URL in the server_address? Having them in two separate fields seems more confusing.

Reply:
I can do that. For the Github Actions workflow for SDK integration tests, SERVER_ADDRESS=localhost:8080 so I thought it would be confusing if the backend integration tests require `http://` or `https://` in the server address but not the SDK integration tests.